target/linux/ramips/image/mt7620.mk

```diff
@@ -193,6 +193,15 @@ define Device/gl-mt300a
 endef
 TARGET_DEVICES += gl-mt300a
 
+define Device/u25awf
+  DTS := U25AWF
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
+  IMAGES += factory.bin
```

Comment:
please drop the IMAGE and IMAGE/factory.bin line. No need to build a factory image that is identical to the sysupgrade image